src/alchemlyb/visualisation/dF_state.py

```diff
@@ -30,7 +30,10 @@ def plot_dF_state(estimators, labels=None, colors=None, units='kcal/mol',
     colors : List
         list of colors for plotting different estimators.
     units : str
-        The unit of the estimate. Default: 'kcal/mol'
+        The unit of the estimate. Default: `kT`
+    scaling_factor : float
+        The scaling factor to change the energy from :math:`kT` to the
+        desired unit.
```

Comment:
remove the scaling factor feature from this PR and make a new one where we can address unit conversion

Btw, I'd call it unit_conversion_factor but perhaps we can come up with a more convenient solution.